src/navigation/BottomTabNavigator.tsx

```diff
@@ -92,11 +96,13 @@ const BottomTabNavigator = (): JSX.Element | null => {
         component={DictionaryStackNavigator}
         options={{ tabBarIcon: renderDictionaryTabIcon, title: getLabels().general.dictionary }}
       />
-      {/*      <Navigator.Screen
-        name='RepetitionTab'
-        component={RepetitionStackNavigator}
-        options={{ tabBarIcon: renderRepetitionTabIcon, title: getLabels().general.repetition }}
-      /> */}
+      {/* <Navigator.Screen */}
```

Comment:
Gah, this makes it so much harder to uncomment again!

Reply:
In IntelliJ you can just use a shortcut and it uncomments this automatically :) For me it is `CTRL + Shift + 7`.

Reply:
This also means that atm this never updates, even if this changes, e.g. by doing the repetition exercise?

Reply:
Hmm, my (un)commenting shortcuts only remove the outermost brackets, but I also use VS Code 🤷  It's also fine, just mildly annoying

Reply:
IntelliJ :heart: